### PR TITLE
File: akka 2.6.

### PR DESCRIPTION
**Pull request by dotbg**
References #2473

### CODE REVIEW

File: file/src/test/java/docs/javadsl/DirectoryTest.java [hunk 1]
```diff
@@ -46,7 +45,7 @@
   @BeforeClass
   public static void beforeAll() throws Exception {
     system = ActorSystem.create();
-    materializer = ActorMaterializer.create(system);
+    materializer = Materializer.apply(system);
```
Comment (seglo) [inline file/src/test/java/docs/javadsl/DirectoryTest.java:48]: We want to drop all the materializer references in the tests (where possible).

**Review by seglo — COMMENTED** [review]
We still need to replace all the materializer references with system in the Java and Scala tests. The easiest way to see if the Akka 2.6 deprecations are fixed is by enabling `fatalWarnings` on the sub project.

File: file/src/test/java/docs/javadsl/DirectoryTest.java (hunk 1 above)
Comment (dotbg) [reply file/src/test/java/docs/javadsl/DirectoryTest.java:48]: @ennru sadly, there is a [method](https://github.com/akka/akka/blob/master/akka-stream-testkit/src/main/scala/akka/stream/testkit/javadsl/StreamTestKit.scala#L18) in the tearDown, requiring the Materializer instance. 

```java
  @After
  public void tearDown() throws Exception {
    fs.close();
    fs = null;
    StreamTestKit.assertAllStagesStopped(materializer);
  }
```

Comment (seglo) [reply file/src/test/java/docs/javadsl/DirectoryTest.java:48]: You can get the materialized from an actor system, `Materializer.matFromSystem(system)`

Comment (dotbg) [reply file/src/test/java/docs/javadsl/DirectoryTest.java]: just pushed it changed

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by ennru** [comment]
Thank you so much for taking these forward!

**Pull request merged**